### ISSUE TITLE
<code> element's background-color problem on IE11 

### ISSUE THREAD

**Issue by PerseusTheGreat** [twbs/bootstrap#16211]
Hi.
I apologize in advance, I am beginner in English
 
Recently, I downloaded Bootstrap 3.3.4 and I was changing its LESS files to creating RTL edition.
Everything was great, but when I was trying to change official examples (exactly sticky-footer-navbar) for testing propose, I found irregular behavior on IE11:

I set <code>dir="rtl"</code> on <code>&lt;html&gt;</code> element
I set <code>style="direction:ltr;"</code> on <code>&lt;code&gt;</code> element, then all contents of <code>&lt;code&gt;</code> element had been hidden! (Why?!)
I set <code>dir="ltr"</code> on <code>&lt;code&gt;</code> element (instead of inline style) but nothing had been changed!

I am using IE11 (11.0.17 - March 2015 Update) on Windows 8.1.
I checked this on Firefox 38 Developer Edition and Chrome 41; this issue occurs only on IE.

I found that, disabling <code>background-color</code> (of <code>&lt;code&gt;</code> element) or overriding it as <code>transparent | inherit</code> made all <code>&lt;code&gt;</code> element contents visible again!

Is this issue consequence of my changes? or it is just a bug of IE or ...?

**Comment by kkirsche** [comment]
Could you post a jsbin of the problem so that we can easily pull it up and test to verify this issue?

**Comment by PerseusTheGreat** [comment]
Sure, <a href="http://jsbin.com/putaji/1/" target="_blank">here</a>.

**Comment by patrickhlauke** [comment]
This looks like a bug in IE to me, as I can't find anything specific to Bootstrap's styling that would cause it (also, Bootstrap doesn't officially support LTR at this point).Testing in BrowserStack, it seems a problem in IE10 and IE11 (strangely, IE8 and IE9 are fine). Reassuringly though, Spartan (future IE replacement) in Win10 seems to have fixed this bug again, so no need to file a bug with MS.

**Comment by nextgenthemes** [comment]
> I was changing its LESS files to creating RTL edition.

Are are aware of https://github.com/morteza/bootstrap-rtl ? Probably no need for you to reinvent the wheel.

**Comment by PerseusTheGreat** [comment]
Thanks to your attentions.

Dear @patrickhlauke 
I found that <code>dispaley:inline-block;</code> can solve my problem but this is not true solution.
In case of long inline <code>&lt;code&gt;</code> content, <code>dispaley:inline-block;</code> cause <code>&lt;code&gt;</code> element acts as block-level elements.

Dear @nextgenthemes 
Nearly whole last year, I have been researched about true and comprehensive RTL layouting via Bootstrap.
I study more than 20 RTL-ized forked/unofficial versions of Bootstrap codes and there was not a general solution.
Some of them use CSS hacks over base Bootstrap CSS codes.
Some of them use JavaScript codes!!!
Some of them are not up to dated.
Some of them are isolated, solid, and unregeneratable.
Many of them, do layouting elements in mirrored LTR situations.
and Many of them are Buggy.
I dont want to reinventing the wheel, but I need a high quality RTL-ized version, like original Bootstrap.
So I tried to build one.
